<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigWindow</class>
 <widget class="QMainWindow" name="ConfigWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>944</width>
    <height>540</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>944</width>
    <height>540</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>944</width>
    <height>540</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnSave">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>420</y>
      <width>88</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>88</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>88</width>
      <height>50</height>
     </size>
    </property>
    <property name="text">
     <string>SAVE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnBack">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>420</y>
      <width>88</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>88</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>88</width>
      <height>50</height>
     </size>
    </property>
    <property name="text">
     <string>BACK</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRST">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>420</y>
      <width>88</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>88</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>88</width>
      <height>50</height>
     </size>
    </property>
    <property name="text">
     <string>RESET</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>901</width>
      <height>80</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>430</width>
      <height>0</height>
     </size>
    </property>
    <property name="title">
     <string>General Config</string>
    </property>
    <widget class="QLineEdit" name="ledNMX">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>30</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="ledNL">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="cbxTFlow">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>30</y>
       <width>210</width>
       <height>31</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>210</width>
       <height>31</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>210</width>
       <height>31</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="lblTFlow">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>30</y>
       <width>151</width>
       <height>34</height>
      </rect>
     </property>
     <property name="text">
      <string>Traffic Flow</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>432</width>
      <height>221</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>432</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>432</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="title">
     <string>Output Files</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_5">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>120</y>
       <width>212</width>
       <height>78</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QLabel" name="lblTFOP">
        <property name="text">
         <string>Time File Output Port</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Switch" name="sbtnTFOP" native="true">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>36</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>36</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>120</y>
       <width>211</width>
       <height>80</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLabel" name="lblSFOP">
        <property name="text">
         <string>Statistic File Output Port</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Switch" name="sbtnSFOP" native="true">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>36</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>36</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>30</y>
       <width>211</width>
       <height>80</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="lblPVOP">
        <property name="text">
         <string>Processed Video Output Port</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Switch" name="sbtnPVOP" native="true">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>36</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>36</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>211</width>
       <height>80</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="lblMVOP">
        <property name="text">
         <string>Mask Video Output Port</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Switch" name="sbtnMVOP" native="true">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>36</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>36</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>350</y>
      <width>432</width>
      <height>131</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>432</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>432</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="title">
     <string>Occlusion Handling Algorithm</string>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>364</width>
       <height>120</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="1" column="0">
         <widget class="Switch" name="sbtnOHAEN" native="true">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>36</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>36</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="lblOHAEN">
          <property name="text">
           <string>Detection</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="lblOHAEN_2">
          <property name="text">
           <string>Solver</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="Switch" name="sbtnOCCSOLVE" native="true">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>36</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>36</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QComboBox" name="cbxOHA">
          <property name="minimumSize">
           <size>
            <width>210</width>
            <height>31</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>210</width>
            <height>31</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>120</y>
      <width>432</width>
      <height>131</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>432</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>432</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="title">
     <string>RGB Mask Extraction From ID</string>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>431</width>
       <height>80</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QLabel" name="lblIDEN">
          <property name="text">
           <string>Enable Track ID Mask</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Switch" name="sbtnIDEN" native="true">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>36</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>36</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLineEdit" name="ledID">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>31</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>31</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>270</y>
      <width>432</width>
      <height>131</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>432</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>432</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="title">
     <string>Additional Geometric Features</string>
    </property>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>-1</x>
       <y>30</y>
       <width>431</width>
       <height>101</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QCheckBox" name="chbTheta">
        <property name="minimumSize">
         <size>
          <width>190</width>
          <height>17</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>190</width>
          <height>17</height>
         </size>
        </property>
        <property name="text">
         <string>Orientation</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="chbPerimeter">
        <property name="minimumSize">
         <size>
          <width>190</width>
          <height>17</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>190</width>
          <height>17</height>
         </size>
        </property>
        <property name="text">
         <string>Perimeter</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="chbEqDiam">
        <property name="minimumSize">
         <size>
          <width>190</width>
          <height>17</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>190</width>
          <height>17</height>
         </size>
        </property>
        <property name="text">
         <string>Equivalent Diameter</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="chbCompact">
        <property name="minimumSize">
         <size>
          <width>190</width>
          <height>17</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>190</width>
          <height>17</height>
         </size>
        </property>
        <property name="text">
         <string>Compactness</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="chbEcc">
        <property name="minimumSize">
         <size>
          <width>190</width>
          <height>17</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>190</width>
          <height>17</height>
         </size>
        </property>
        <property name="text">
         <string>Eccentricity</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="chbConvex">
        <property name="minimumSize">
         <size>
          <width>190</width>
          <height>17</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>190</width>
          <height>17</height>
         </size>
        </property>
        <property name="text">
         <string>Convexity</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>944</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Switch</class>
   <extends>QWidget</extends>
   <header location="global">Animator.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
